tests/om.spec.ts

```diff
@@ -0,0 +1,9 @@
+import generateCMPTests from "../playwright/runner";
+
+generateCMPTests('om', [
+  'https://olli-machts.de/en/extension/cookie-manager',
+  'https://www.uni-assist.de/',
+  'https://www.unisabana.edu.co/',
+  'https://deutsche-alzheimer.de/',
```

Comment:
Testing on these websites I saw the following issues:
1. The popup comes back on every page load. This even happens when I do the opt-out manually, so seems to be an issue with the popup itself.
2. Sometimes the popup stays after the opt-out completes. This suggests that we're hitting a race condition where we hit the button too fast for the popup to handle it.

Reply:
It was indeed a race condition 🙏 I changed the selectors, and not it seems to be more stable